### PR TITLE
Improve packages docs references and links

### LINKED ISSUE #1376: Improve documentation by shipping READMEs in Pypi/NPM
In order to make it easier for everyone, we should just package some README in those. Right now, Pypi landing pages and NPMJS landing pages are showing some unintuitive "No README".

### PR DESCRIPTION
**Pull request by lissyx**
Fixes #1376

### CODE REVIEW

File: native_client/Makefile [hunk 1]
```diff
@@ -31,7 +31,7 @@ bindings-clean:
 	rm -f python/model_wrap.cpp python/utils_wrap.cpp python/model.py python/utils.py
 
 bindings-build:
-	pip install --quiet $(PYTHON_PACKAGES) wheel
+	pip install --quiet $(PYTHON_PACKAGES) wheel==0.31.0 setuptools==39.1.0
```
Comment (lissyx) [inline native_client/Makefile]: Required to ensure proper handling of README as markdown.

File: native_client/javascript/Makefile [hunk 2]
```diff
@@ -19,6 +19,9 @@ clean-npm-pack:
 really-clean: clean clean-npm-pack
 	rm -fr ./lib/
 
+README.md:
+	cp ../../README.md README.md
```
Comment (lissyx) [inline native_client/javascript/Makefile]: Symlinking does not work with npm, of course :)

File: native_client/setup.py [hunk 3]
```diff
@@ -17,6 +17,9 @@
     numpy_include = ''
     assert 'NUMPY_INCLUDE' in os.environ
 
+def read(fname):
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
```
Comment (reuben) [inline native_client/setup.py]: This is meant to handle cases where setup.py is called from a different directory, right?

File: native_client/setup.py [hunk 4]
```diff
@@ -49,13 +52,22 @@ class BuildExtFirst(build):
 
 setup(name = project_name,
       description = 'A library for running inference on a DeepSpeech model',
+      long_description = read('../README.md'),
+      long_description_content_type = 'text/markdown; charset=UTF-8',
```
Comment (reuben) [inline native_client/setup.py]: Nice :)

File: native_client/javascript/package.json.in [hunk 5]
```diff
@@ -7,6 +7,17 @@
        "deepspeech": "./client.js"
     },
     "author" : "Mozilla",
+    "license": "MPL-2.0",
+    "homepage": "https://github.com/mozilla/DeepSpeech/tree/$(PROJECT_VERSION)#project-deepspeech",
+    "files": [
```
Comment (reuben) [inline native_client/javascript/package.json.in]: Is this required? Would prefer to not add a list that needs to be updated every time file names/locations change if that's possible.

File: native_client/setup.py (hunk 3 above)
Comment (lissyx) [reply native_client/setup.py]: Yep. It's even what they use in the doc

File: native_client/javascript/package.json.in (hunk 5 above)
Comment (lissyx) [reply native_client/javascript/package.json.in]: We can remove that, but then we end up packaging a lot of useless elements. Up to you, though :)

**Pull request merged**